adform-htb.js

```diff
@@ -144,9 +218,36 @@ function AdformHtb(configs) {
         var queryObj = {};
         var callbackId = System.generateUniqueId();
 
-        /* Change this to your bidder endpoint.*/
-        var baseUrl = Browser.getProtocol() + '//someAdapterEndpoint.com/bid';
+        var request = [], item, _key, _value, i, j, k;
 
+        var globalParams = [ [ 'adxDomain', 'adx.adform.net' ], [ 'fd', 1 ], [ 'url', null ], [ 'tid', null ] ];
+        for (i in returnParcels) {
+
+            item = returnParcels[i].xSlotRef;
+            for (j = 0, k = globalParams.length; j < k; j++) {
+                _key = globalParams[j][0];
+                _value = item[_key];
+                if (_value) {
+                  item[_key] = null;
+                  globalParams[j][1] = _value;
+                }
+            }
+
+            request.push(getRequestItem(item));
+        }
+
+        request.unshift(Browser.getProtocol() + '//' + globalParams[0][1] + '/adx/?rp=4');
+
+        for (i = 1, k = globalParams.length; i < k; i++) {
```

Comment:
Similar comments above, is there a need for the loop?

Reply:
Moved global request params configuration out of slots config. Is that ok?